<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1536314791196' stop='1536314797638'>
    <name>Automation BankManager Login functionality</name>
    <title>Automation BankManager Login functionality</title>
    <test-cases>
        <test-case start='1536314791196' status='passed' stop='1536314797202'>
            <name>Login to the Bank Manager account</name>
            <title>Login to the Bank Manager account</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1536314797202' status='failed' stop='1536314797637'>
            <name>Validate Add Customer</name>
            <title>Validate Add Customer</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: No element found using locator: by.model("postCd)")</message>
                <stack-trace>NoSuchElementError: No element found using locator: by.model("postCd)")
                    at elementArrayFinder.getWebElements.then (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27)
                    at ManagedPromise.invokeCallback_ (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at asyncRun (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
                    at C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at &lt;anonymous>
                    at process._tickCallback (internal/process/next_tick.js:188:7)Error
                    at ElementArrayFinder.applyAction_ (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.(anonymous function).args [as sendKeys] (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.(anonymous function).args [as sendKeys] (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
                    at UserContext.&lt;anonymous> (D:\Practice-protractor\rough\BankManagerLogin.js:23:38)
                    at C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                From: Task: Run it("Validate Add Customer") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
                    at Spec.execute (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
                    at UserContext.fn (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
                    at attempt (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at runNext (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (D:\Practice-protractor\rough\BankManagerLogin.js:17:5)
                    at addSpecsToSuite (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\ddminh\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (D:\Practice-protractor\rough\BankManagerLogin.js:5:1)
                    at Module._compile (module.js:652:30)
                    at Object.Module._extensions..js (module.js:663:10)
                    at Module.load (module.js:565:32)
                    at tryModuleLoad (module.js:505:12)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>